### ISSUE TITLE
Expanding log output to get the loss of the ClassID while training

### ISSUE THREAD

**Issue by flowzen1337** [AlexeyAB/darknet#6057]
Hi @AlexeyAB,

i guess that is a easy one :-)

So while training, i can seek through the log file and get much informations about the whole dataset and the average loss of my currently trained model.
But what i'm missing is the indicator to which class each line in the log belongs to.

For example that's are three lines from my current model which i'm training, so far so good:

`...`
`...`
`v3 (iou loss, Normalizer: (iou: 0.07, cls: 1.00) Region 150 Avg (IOU: 0.741034, GIOU: 0.731050), Class: 0.937038, Obj: 0.170851, No Obj: 0.001736, .5R: 1.000000, .75R: 0.500000, count: 14, class_loss = 39.437553, iou_loss = 238.848480, total_loss = 278.286041`
`v3 (iou loss, Normalizer: (iou: 0.07, cls: 1.00) Region 161 Avg (IOU: 0.765446, GIOU: 0.760826), Class: 0.756056, Obj: 0.440192, No Obj: 0.003780, .5R: 1.000000, .75R: 0.428571, count: 7, class_loss = 15.896358, iou_loss = 15.458718, total_loss = 31.355076`
`v3 (iou loss, Normalizer: (iou: 0.07, cls: 1.00) Region 139 Avg (IOU: 0.000000, GIOU: 0.000000), Class: 0.000000, Obj: 0.000000, No Obj: 0.000112, .5R: 0.000000, .75R: 0.000000, count: 1, class_loss = 0.688350, iou_loss = 0.000000, total_loss = 0.688350`
`...`
`...`

I see the section "Class: 0.937038" and "class_loss: 39.437553", but i can't link it to the ClassID the line belongs to.

So my idea is, to also add a section "ClassID: " to the logfile lines, so that you know the current loss of the individual classes while training.
That would help to get a indicator which Class isn't training so well and that will than help improving the dataset as you then know which class needs to get more annotations (or other improvements) to help improving your dataset as you got the Loss and the ClassID then.

I guess that's just one line of code to add somewhere in the yolo sourcecode and wouldn't make much work for you to add that also to the log. I guess also in the chart which is provided when using the "-map" flag, it would be a nice feature to see the loss while training for each individual ClassID.

Thanks for your help here! :-)
